CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## v1.8
 
+### Continuous integration
+
+* GitHub Actions CI workflows are now included in the template pipeline
```

Comment:
```suggestion
* GitHub Actions CI workflows are now included in the template pipeline
  * Please update these files to match the existing tests that you have in `.travis.yml`
```

I wonder if we should add some `TODO` statements in the CI workflow. Then the linting will throw a warning if it's not customised.

Reply:
https://github.com/nf-core/tools/pull/488

Reply:
Ive committed a fix so when tests pass Ill merge 👍